.github/workflows/terraform_provider.yml

```diff
@@ -256,3 +258,77 @@ jobs:
           -provider-name aws \
           -providers-schema-json terraform-providers-schema/schema.json \
           -require-resource-subcategory
+
+  # See https://github.com/actions/upload-release-asset
+  babbel_release:
+    name: Release Babbel fork
+    needs: [go_test, golangci-lint] 
+    if: startsWith( github.event.ref, 'refs/tags/v' )
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: set up Go 1.14
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14
+
+    - name: check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
+            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+            dep ensure
+        fi
+    - name: parse version string
+      id: parse_version
+      run: |
+        export BUILD_VERSION=$(echo $GITHUB_REF | sed -e 's/refs.tags.\(.*\)$/\1/')
+        export BUILD_NAME=terraform-provider-aws_$BUILD_VERSION
+        echo "::set-output name=BUILD_NAME::$BUILD_NAME"
+        echo ::set-output name=ZIP_NAME_LINUX::"$BUILD_NAME"_linux_amd64.zip
+        echo ::set-output name=ZIP_NAME_DARWIN::"$BUILD_NAME"_darwin_amd64.zip
+    - name: build Linux
```

Comment:
As currently this is duplicating all the parts for linux and Darwin I would suggest to use `strategy` which works similar to the matrix in travis. You can then define the two strategies and run the build for both of them instead of duplicating the steps https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy

Reply:
@fmasuhr I have created an alternative implementation using the matrix feature. However, it introduces a lot of overhead. Please have a look and let me know which version you think is better: https://github.com/babbel/terraform-provider-aws/pull/34